### PR TITLE
Add support for Netbox metrics configurations

### PR DESCRIPTION
**Pull request by tyler-8**
Fixes https://github.com/lae/ansible-role-netbox/issues/68

Adds configuration and templates for metrics configurations.

### CODE REVIEW

**Comment by tyler-8** [comment]
Had to tweak the `uwsgi` template slightly. Ran the full playbook again on a RHEL7 box and all the pieces were there, `/metrics` worked as intended.

**Comment by lae** [comment]
Yeah, I don't doubt it works as intended, but it should still be enabled for the CI tests so that if it does break in any significant manner, it'll be caught.

**Comment by tyler-8** [comment]
Oh my bad, I misunderstood what you said before. Will do!

File: tests/test.yml [hunk 1]
```diff
@@ -25,3 +25,6 @@
     - name: Ensure that NetBox returns a successful HTTP response
       uri:
         url: "http://{{ inventory_hostname }}:8080"
+    - name: Check that Netbox metrics endpoint works
+      uri:
+        url: "http://{{ inventory_hostname }}:8080/metrics
```
Comment (lae) [inline tests/test.yml]: Closing quotation mark is missing here.

**Pull request merged**